htlcswitch/link.go

```diff
@@ -1109,19 +1112,10 @@ out:
 			}
 
 		case <-l.cfg.BatchTicker.Ticks():
-			// If the current batch is empty, then we have no work
```

Comment:
Nice, this should also lower idle CPU usage for routing nodes with tons of channels.